components/director/pkg/graphql/schema.graphql

```diff
@@ -1,552 +1,532 @@
-# Directives
+directive @hasScopes(path: String!) on FIELD_DEFINITION
+scalar Any
 
-directive @hasScopes(path:String!) on FIELD_DEFINITION
-# Scalars
+scalar CLOB
 
-scalar Any # -> interface{}
+scalar HttpHeaders
```

Comment:
What about our comments? Can we keep them somehow?

Reply:
No sir, only graphql comments are preserved